lib/xlat_tables_v2/xlat_tables_core.c

```diff
@@ -329,7 +329,10 @@ static void xlat_tables_unmap_region(xlat_ctx_t *ctx, mmap_region_t *mm,
 			xlat_tables_unmap_region(ctx, mm, table_idx_va,
 						 subtable, XLAT_TABLE_ENTRIES,
 						 level + 1U);
-
+#if !HW_ASSISTED_COHERENCY
```

Comment:
I think this needs to be `#if !(HW_ASSISTED_COHERENCY || WARMBOOT_ENABLE_DCACHE_EARLY)`. See https://github.com/ARM-software/arm-trusted-firmware/pull/1498

Reply:
Done.